etc/local_grapl/bin/upload-osquery-logs.py

```diff
@@ -32,6 +32,17 @@ def hack_PATH_to_include_grapl_tests_common() -> Callable:
     return upload_osquery_logs
 
 
+def setup_env(bucket_prefix: str):
```

Comment:
If you're curious why we do this with `os.putenv`s, please read the requirements on line 15.
(sidenote - i hate this requirement and would love to remove it when it makes sense to do so)